packages/token-balance-cache/test/erc1155.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { assert, expect, use as chaiUse } from 'chai';
+import { BalanceType } from 'common-common/src/types';
+import Web3 from 'web3';
+import chaiAsPromised from 'chai-as-promised';
+
+chaiUse(chaiAsPromised);
+
+import type { IChainNode } from '../src/types';
+import { BalanceProvider } from '../src/types';
+
+class MockErc1155BalanceProvider extends BalanceProvider<{
```

Comment:
I'm not sure what these tests are actually supposed to be testing? Can you describe / elaborate?

Reply:
My idea was to just to test the input verification checks for calling getBalance on the erc1155 balance provider.